static/js/core.js

```diff
@@ -1,226 +1,89 @@
-        if (Y.one('.tabbed-content')) {
-            Y.one('.tabbed-menu').append('<img src="//assets.ubuntu.com/sites/ubuntu/latest/u/img/patterns/tabbed-nav-arrow.png" class="arrow" height="6" width="12" alt="">');
+core.svgFallback = function() {
+	if (typeof Modernizr === "object") {
+		if (!Modernizr.svg || !Modernizr.backgroundsize) {
+			var svgs = document.querySelectorAll("img[src$='.svg']")
+            svgs.each(function(node) {
+				node.src = node.src + '?fmt=png';
+			});
+		}
+	}
+};
 
-            var p = Y.one('.tabbed-menu a.active'),
-                s = p.get('href').split('#')[1],
-                a = Y.one('.arrow'),
-                w = (p.get('clientWidth') / 2) - 7,
-                x = (p.get('parentNode').getXY()[0] - p.get('parentNode').get('parentNode').getXY()[0]) + w;
-            Y.all('.tabbed-content').each(function () {
-                if (this.get('id') !== s) {
-                    this.setStyle('opacity', '0');
-                }
-            });
-            a.setStyle('left', x + 'px').setStyle('display', 'inline');
-            Y.all('.tabbed-menu a').on('click', function (e) {
-                e.preventDefault();
-                Y.all('.tabbed-menu a').removeClass('active');
-                e.currentTarget.addClass('active');
-                Y.all('.tabbed-content').addClass('hide').setStyle('opacity', '0');
-                s = e.currentTarget.get('hash');
-                Y.one(s).removeClass('hide');
-                new Y.Anim({ node: s, to: { opacity: 1 } }).run();
-                x = (e.currentTarget.get('parentNode').getXY()[0] - e.currentTarget.get('parentNode').get('parentNode').getXY()[0]) + w;
-                new Y.Anim({ node: a, to: { left: x + 'px' } }).run();
-            });
-        }
-    };
+core.mobileNav = function () {
+    var header = document.querySelector('header.banner');
+    var navPrimary = document.querySelector('.nav-primary');
+    var navSecondary = document.querySelector('.nav-secondary');
+    var headerSearch = document.querySelector('.header-search');
+    var breadcrumbLinks = document.querySelector('.breadcrumb li a');
 
-    // REMOVE UNSUED
-    core.tabbedContent = function () {
-        Y.all('.tabbed-content .accordion-button').on('click', function (e) {
-            e.preventDefault();
-            e.target.get('parentNode').toggleClass('open');
-        });
-    };
-    
-    // NEED ... WRITE IN ES5
-	core.svgFallback = function() {
-		if (typeof Modernizr === "object") {
-			if (!Modernizr.svg || !Modernizr.backgroundsize) {
-				Y.all("img[src$='.svg']").each(function(node) {
-					src = node.getAttribute('src').toString();
-					if (src.indexOf('assets.ubuntu.com/v1/') > -1) {
-						// Support for the newer asset server
-						src = src + '?fmt=png';
-					} else {
-						// Old asset manager and locally assets
-						src = src.match(/.*\/(.+?)\./)[0] + 'png';
-					}
-					node.setAttribute("src", src);
-				});
-			}
-		}
-	};
-  // BAD, USE CSS
```

Comment:
In fact the resource box has been totally rebuilt and BEMed so there are no `.resource` classes anyway.